src/Nest/QueryDsl/TermLevel/Wildcard/WildcardQuery.cs

```diff
@@ -10,7 +10,7 @@ namespace Nest
 	public interface IWildcardQuery : ITermQuery
 	{
 		[JsonIgnore]
-		[Obsolete("Use MultiTermQueryRewrite")]
+		[Obsolete("Use MultiTermQueryRewrite, deleted in NEST 6.x")]
```

Comment:
I think `fixed` is a better description here; The property `Rewrite` exists in NEST 6.x, but it is of type `MultiTermQueryRewrite`.